website/src/content/blog/2025-03-31-typespec-1-0-release/typespec_1_0_release.md

```diff
@@ -0,0 +1,215 @@
+---
+slug: 2025-03-31-typespec-1-0-release
+title: "TypeSpec 1.0-RC: Write your API once, let TypeSpec do the rest"
+image: ./TypeSpec_1_0_RC.png
+description: "The TypeSpec 1.0 Release Candidate is here! This open-source project, created by Microsoft, brings a powerful yet concise API modeling language that generates server code, client libraries, schemas, and documentation from a single source of truth."
+publishDate: 2025-03-19
+author:
+  name: Mario Guerra
+  title: Senior Product Manager @ Microsoft
+authorAvatar: assets/img/authors/mario_guerra.png
+tags:
+  - release
+  - announcement
+---
+
+We're excited to announce the TypeSpec 1.0 Release Candidate, marking a significant milestone for the TypeSpec project. TypeSpec embodies a simple yet powerful principle: write your API once, and let TypeSpec take care of the rest. This straightforward approach allows you to focus on designing the perfect API contract while TypeSpec handles generating all the artifacts you need - from OpenAPI specifications to client libraries and server code scaffolding - saving you time and eliminating inconsistencies.
+
+## What is TypeSpec?
+
+TypeSpec is a language for describing API contracts. Using TypeSpec, you write your API definition in a concise, human-readable format, and TypeSpec takes care of generating:
+
+- API specifications (OpenAPI, JSON Schema, Protocol Buffers)
+- Server-side code skeletons (C#, JavaScript)
```

Comment:
Perhaps separate out the concepts from the point in time limitations like C# and JS only TODAY. So readers get a clear understanding of the overall goals and vision versus what is actually working today.